account-kit/smart-contracts/src/light-account/clients/multiOwnerLightAccount.ts

```diff
@@ -36,6 +43,30 @@ export type CreateMultiOwnerLightAccountClientParams<
     "transport" | "account" | "chain"
   >;
 
+export type CreateAccountParams<
```

Comment:
nit: can we explicitly include LightAccount in this name? also it looks like these params are client params rather than account params